### PR TITLE
financial algorithm post

### PR DESCRIPTION
**Pull request by TheKharleeci**
Changes:

- Add Olli's Financial algorithms blog post
- Add new Images ( This will still be used on the home page)
- Edit FiQCI bio- It basically said "We build FiQCI" before
- Remove "Move to top" sign ( Looking for an alternative or to make it cleaner)

### CODE REVIEW

File: _posts/2024-04-09-Financial-Algorithms.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+Last November, the Hanken School of Economics in Finland and Ultrahack organized a quantum hackathon for finance-oriented minds with the theme of sustainable finance in the quantum era. The competition saw many teams with different backgrounds tackling the challenge and coming up with innovative solutions to combine the financial sector with the capabilities of quantum computers. Taking part in supporting the event, CSC, with colleagues from VTT and IQM, provided participants access to the 20-qubit Leena quantum computer through the LUMI supercomputer.
+
+It was great to see teams utilizing the Finnish quantum computer as a part of their projects. One such team, Qumpula Quantum, won 2nd place in the competition. A popular topic in this year's hackathon was quantum machine learning, which was also implemented using Leena.
+
+<figure>
+    <img src="/assets/images/Financial-Algorithms-Blog/Figure5-Hanken-Hackathon.png" alt="Figure 5: Hanken-Hackathon">
+    <figcaption>
+    <p>
+    <em>Figure 5: CSC had a team mentoring at Hanken Quantum Hackathon. Pictured at the bottom left: Nikolas, Olli, Modupe and Jake.</em>
```
Comment (JMuff22) [inline _posts/2024-04-09-Financial-Algorithms.md:90]: I would suggest full names

**Comment by JMuff22** [comment]
I would prefer to add images along with home page changes in a separate pull request.

**Comment by JLenssen** [comment]
- The blog post titles are currently underlined in `/publications` can we remove the lines?
- Pictures in blog post overview page are quite small. Can we make them slightly larger?
- `Watch this space for the latest news and posts of interest!` <= imo not needed. It's clear what this page is about.

File: _posts/2024-04-09-Financial-Algorithms.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+Now is the time to identify algorithms with future potential. Many applications have already been singled out: cryptography, simulations, search algorithms, and optimization problems. In finance, the use of quantum algorithms is investigated in asset pricing, risk analysis, and portfolio optimization, to mention a few [2,3]. One promising quantum algorithm for financial analysis is called quantum amplitude estimation. It provides quadratic speed-up over currently known classical methods, a significant advantage in sufficiently large calculations.
```
Comment (JLenssen) [inline _posts/2024-04-09-Financial-Algorithms.md:18]: Any chance we can make the links clickable?

File: _posts/2024-04-09-Financial-Algorithms.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+
+## Quantum amplitude estimation
+Quantum amplitude estimation (QAE) was originally developed by Brassard et al. [1]. It is a generalization of the famous Grover’s search algorithm and can be used to find the amplitude of a quantum state. It uses quadratically fewer queries than classical sampling methods. Thus, what would require a million queries using purely classical methods can ideally be done with just a thousand quantum queries. A hundred million classical queries correspond to only ten thousand quantum queries, and so on.
+
+The impatient Reader, who just wants to know how all of this works in practice, can skip to the next section. The technically inclined should read on.
```
Comment (JLenssen) [inline _posts/2024-04-09-Financial-Algorithms.md:26]: Reader -> reader

File: _posts/2024-04-09-Financial-Algorithms.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+## Notebooks
+Those interested in the codes used for this blog can find the Jupyter notebooks with detailed explanations in the link below. They can be executed directly on the FiQCI infrastructure. [More details here](https://github.com/CSCfi/Quantum/tree/main/Finance-Modelling-on-Helmi)
+
+## References
+
+1.	[Quantum Amplitude Amplification and Estimation, G. Brassard et al., 2000](https://arxiv.org/abs/quant-ph/0005055)
+2.	[Option Pricing using Quantum Computers, N. Stamatopoulos et al., 2000](https://arxiv.org/abs/1905.02666)
+3.	[Quantum Risk Analysis, S. Woerner and D.J. Egger, 2018](https://arxiv.org/abs/1806.06893)
+4.	[Amplitude Estimation without Phase Estimation, Y. Suzuki et al., 2020](https://arxiv.org/abs/1904.10246)
+5.	[Improved maximum-likelihood quantum amplitude estimation, A. Callison, D. Browne, 2023](https://arxiv.org/abs/2209.03321)
```
Comment (JLenssen) [inline _posts/2024-04-09-Financial-Algorithms.md:109]: A. Callison _and_ D. Browne

**Comment by olmuk** [comment]
> 



> * The blog post titles are currently underlined in `/publications` can we remove the lines?
> * Pictures in blog post overview page are quite small. Can we make them slightly larger?
> * `Watch this space for the latest news and posts of interest!` <= imo not needed. It's clear what this page is about.

@JLenssen Do you mean all of the pictures? We can make them larger

**Review by JLenssen — APPROVED** [review]

File: _posts/2024-04-09-Financial-Algorithms.md [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+    <p>
+    <em>Figure 4: Statistical mean of amplitude error for evaluating the European call option. Comparison between MLQAE algorithm on a simulator and the Helmi quantum computer, and a classical sampling method</em>
+    </p>
+    </figcaption>
+</figure>
+
+## Finance and quantum computers gather public interest
+Experts from different fields interested in gaining a quantum advantage in the future are now working to improve their knowledge and encouraging people to learn about quantum technologies.
+
+Last November, the Hanken School of Economics in Finland and Ultrahack organized a quantum hackathon for finance-oriented minds with the theme of sustainable finance in the quantum era. The competition saw many teams with different backgrounds tackling the challenge and coming up with innovative solutions to combine the financial sector with the capabilities of quantum computers. Taking part in supporting the event, CSC, with colleagues from VTT and IQM, provided participants access to the 20-qubit Leena quantum computer through the LUMI supercomputer.
```
Comment (olmuk) [inline _posts/2024-04-09-Financial-Algorithms.md:82]: "Taking part in supporting the event, CSC with colleagues from VTT and IQM " -> "Taking part in supporting the event, CSC and VTT with colleagues from IQM "

**Comment by olmuk** [comment]
Let's add the name of the author. Also, let's see if we can add a picture of the author

File: _layouts/posts.html [hunk 6]
```diff
@@ -34,6 +34,6 @@ <h2 class="archive__subtitle">{{ }}</h2>
         {% endfor %}
       {% endfor %}
     </div>
-    <a href="#page-title" class="back-to-top">{{ site.data.ui-text[site.locale].back_to_top | default: 'Back to Top' }} &uarr;</a>
+    <!-- <a href="#page-title" class="back-to-top">{{ site.data.ui-text[site.locale].back_to_top | default: 'Back to Top' }} &uarr;</a> -->
```
Comment (JMuff22) [inline _layouts/posts.html:37]: Lets remove this instead of commenting it out

File: _pages/publications.md [hunk 7]
```diff
@@ -17,7 +17,7 @@ intro:
 
 <!-- {% include feature_row id="intro" type="center" %} -->
 
-Watch this space for the latest news and posts of interest!
+<!-- Watch this space for the latest news and posts of interest! -->
```
Comment (JMuff22) [inline _pages/publications.md:20]: Lets remove this instead of commenting it out

**Review by JMuff22 — CHANGES_REQUESTED** [review]

**Review by olmuk — APPROVED** [review]

**Pull request merged**